models/articles.js

```diff
@@ -46,4 +48,21 @@ function selectComments(article_id) {
     });
 }
 
-module.exports = { selectArticles, selectArticle, selectComments };
+function insertComment(newComment, article_id) {
+  const formattedComment = prepareNewComment(newComment, article_id);
+  const queryString = format(
```

Comment:
Any reason you're using `pg-format` here and not just `$1` from `pg`?

Reply:
No reason, I forgot that was just one comment being added so no need to use pg-format will update